### DISCUSSION TITLE
Pasting images

### DISCUSSION THREAD

**Discussion by widle** [asciidocfx/AsciidocFX#564] [Q&A] [answered]
Hello, I have a question about the feature that allows pasting an image from clipboard to the doc (#219). Is there a way to set up the target directory? It seems to automatically paste to /images and also appends images/ to the path pasted in the document, but that only works if the image directory is the default one (same as the doc). I'd like to set an imagesdir for my document and paste into that, but setting the attribute does not change the pasting behaviour.

I have found the setting to change the generated image name, but not the folder. Am I missing something, or is it not implemented?

**Comment by rahmanusta** [comment, marked as answer]
Hello,

This is something not implemented yet. I consider to enhance this feature but need some time to do that.

**Reply by widle** [reply]
Thank you. I'll make this a feature request then.
